v3/non-airgap/database-preflight.yml

```diff
@@ -0,0 +1,81 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: database-preflights
+  version: 0.0.2
+spec:
+  hostCollectors:
+    - cpu: {}
+    - memory: {}
+    - hostOS: {}
+    - diskUsage:
+        collectorName: root-disk
+        path: /
+    - run:
+        collectorName: "PreflightVersion"
+        command: "echo"
+        args: [ "Preflight version - 0.0.2" ]
```

Comment:
could you increase this version to 0.0.3 please?

Reply:
good call